src/main/java/mil/dds/anet/resources/AttachmentResource.java

```diff
@@ -218,9 +224,9 @@ private void assertAllowedMimeType(final String mimeType) {
     }
   }
 
-  private void assertAllowedClassification(Attachment.Classification classification) {
-    if (DaoUtils.getEnumId(classification) != 0) {
-      // classification is not "undefined"
+  private void assertAllowedClassification(String classification) {
+    if (classification != null) {
+      // classification is not null
       throw new WebApplicationException("Classification cannot be set", Status.FORBIDDEN);
```

Comment:
This status code has a different purpose https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406

I think https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400 should be use if you do not agree with the input. 

Why can the classification not be `null`?

Reply:
Classification does not has to be null on create from API point of view.